biztalk/core/how-to-configure-transport-advanced-options-for-a-send-port.md

```diff
@@ -23,6 +23,13 @@ manager: "anneta"
 # How to Configure Transport Advanced Options for a Send Port
 Use the BizTalk Server Administration console to configure transport advanced options for a send port. These options determine how messages are handled by the send port, such as the number of times to retry sending messages on message failure and the service window schedule for the port.  
   
+> [!NOTE]
+> **Starting with BizTalk Server 2020**, dynamic send ports with ordered delivery can process messages to different outbound locations in parallel, allowing for higher throughput. Order will be maintained per unique outbound location, but not across different outbound locations even for the same transport type.
```

Comment:
Wondering if this note should go after the 2016 ability to allow ordered delivery in dynamic send port. 
Chronology in reading order?